android/src/main/java/com/magicleap/magicscript/scene/nodes/video/VideoNode.kt

```diff
@@ -91,6 +106,38 @@ class VideoNode(
         setAction(props)
         setLooping(props)
         setVolume(props)
+        setSeekTo(props)
+        if(props.containsAny(PROP_SIZE, PROP_LOCAL_SCALE, PROP_LOCAL_POSITION)) updateSubtitlesProps()
```

Comment:
Use brackets after the `if` and put `updateSubtitlesProps()` into new line (we followed this practice so far)